plugin-pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PinerySource.java

```diff
@@ -693,6 +692,7 @@ private static long runLaneCount(RunDto dto) {
   private Optional<PineryConfiguration> config = Optional.empty();
   private final PlatformCache platforms;
   private final ProjectCache projects;
+  private HttpClient httpClient = null;
```

Comment:
This will be constantly rebuilt and replaced, so keeping this reference to the latest one is probably going to be more confusing than useful. Is it needed for some reason? If so, rebuilding/replacing it is probably going to interfere with that.

Reply:
My bad - didn't notice the null check in `httpClient()`